### DISCUSSION TITLE
absence of CDR3 parts vs lack of a clone sequence?

### DISCUSSION THREAD

**Discussion by Angieiscute** [milaboratory/mixcr#1425] [Q&A] [answered]
Hello,

I am curious about the difference between the message "Alignment failed: no CDR3 parts" in the Alignment log and "Reads dropped due to the lack of a clone sequence" in the Assembly log.

I am puzzled as to why there are still some reads being dropped during the assembly process due to the absence of a clone sequence (no CDR3), even though these reads should have already been excluded during the alignment process ("Alignment failed: no CDR3 parts").

Thank you!

**Comment by mizraelson** [comment, marked as answer]
Hi,
It happens sometimes when a part of CDR3 is covered, then it can pass the alignments filter, but later fails to assemble the full sequence. It's easier to examine such things by visually inspecting the alignments using [`mixcr exportAlignmentsPretty`](https://mixcr.com/mixcr/reference/mixcr-exportPretty/).

**Reply by Angieiscute** [reply]
Thanks!
How about "Reads dropped due to failed mapping" in the assemble log? What's the difference between "Alignment failed: no hits"?

**Reply by mizraelson** [reply]
Reads dropped due to failed mapping - low quality reads, that failed to map to core clones.
Alignment failed: no hits - non target reads, that didn't align to the reference sequence.
